pom.xml

```diff
@@ -76,8 +76,7 @@ THE SOFTWARE.
     <properties>
         <revision>1.43</revision>
         <changelist>-SNAPSHOT</changelist>
-        <powermock.version>1.6.3</powermock.version>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.89.4</jenkins.version>
```

Comment:
This is still pretty old. @thoulen any preferences on a minimum version?

Reply:
@jvz I updated it to `2.121.1` along with a bunch of test deps to make the java 11 tests pass